### PR TITLE
patch for CVE-2020-15228

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

File: .github/workflows/mvn-build.yml [hunk 1]
```diff
@@ -13,11 +13,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2.3.4
```
Comment (gsmet) [inline .github/workflows/mvn-build.yml:16]: Are you sure this is a good idea? I would expect the GH Actions to be updated automatically?

File: .github/workflows/mvn-build.yml [hunk 2]
```diff
@@ -13,11 +13,13 @@ jobs:
...
       - name: Set up JDK 11
         uses: joschi/setup-jdk@7134ae95986e4e0a4f9f0b51345c93fcebfc4ea9
         with:
           java-version: openjdk11
       - name: Build
-        run: mvn clean install
+        run: |
+          echo "JAVA_HOME=/opt/hostedtoolcache/AdoptOpenJDK/1.0.0-releases-openjdk11-hotspot-normal-latest/x64" >> $GITHUB_ENV
```
Comment (gsmet) [inline .github/workflows/mvn-build.yml:24]: I think you should just update `joschi/setup-jdk` to the version we use in Quarkus: `e87a7cec853d2dd7066adf837fe12bf0f3d45e52`

**Review by gsmet — CHANGES_REQUESTED** [review]

File: .github/workflows/mvn-build.yml (hunk 2 above)
Comment (gsmet) [reply .github/workflows/mvn-build.yml:24]: At least we didn't have to do any of this on Quarkus.

File: .github/workflows/mvn-build.yml (hunk 1 above)
Comment (aureamunoz) [reply .github/workflows/mvn-build.yml:16]: Just wanted to do a test. I should have done a Draft PR

File: .github/workflows/mvn-build.yml (hunk 2 above)
Comment (aureamunoz) [reply .github/workflows/mvn-build.yml:24]: yes, I have taken the Quarkus one, just didn't have the time to push. Thank you!!

**Comment by aureamunoz** [comment]
I think I can merge now, wdyt @gsmet ?

**Pull request merged**